packages/cosmwasm-stargate/src/signingcosmwasmclient.spec.ts

```diff
@@ -1195,6 +1195,44 @@ describe("SigningCosmWasmClient", () => {
 
         client.disconnect();
       });
+
+      it("works with a custom timeout height", async () => {
+        pendingWithoutWasmd();
+        const wallet = await DirectSecp256k1HdWallet.fromMnemonic(alice.mnemonic, { prefix: wasmd.prefix });
+        const client = await SigningCosmWasmClient.connectWithSigner(wasmd.endpoint, wallet, {
+          ...defaultSigningClientOptions,
+        });
+
+        const msg = MsgSend.fromPartial({
+          fromAddress: alice.address0,
+          toAddress: alice.address0,
+          amount: [coin(1, "ucosm")],
+        });
+        const msgAny: MsgSendEncodeObject = {
+          typeUrl: "/cosmos.bank.v1beta1.MsgSend",
+          value: msg,
+        };
+        const fee = {
+          amount: coins(2000, "ucosm"),
+          gas: "222000", // 222k
+        };
+        const memo = "Use your power wisely";
+        const height = await client.getHeight();
+        const signed = await client.sign(
+          alice.address0,
+          [msgAny],
+          fee,
+          memo,
+          undefined,
+          Long.fromNumber(height + 1),
+        );
+
+        // ensure signature is valid
+        const result = await client.broadcastTx(Uint8Array.from(TxRaw.encode(signed).finish()));
+        assertIsDeliverTxSuccess(result);
```

Comment:
Could you add a test (in both cases) where the timeout height is exceeded? You can use `client.getHeight()-1` or something like that for the timeout height.

Reply:
Nice updates. As far as I can see only those thests are missing: timeout height exceeded.

Reply:
exactly, just added those